dev-docs/bidders/viewdeosDX.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: bidder
+title: ViewDeos DX
+description: Prebid ViewDeos Bidder Adaptor
+hide: true
+biddercode: viewdeosDX
+media_types: video
+---
+
+### Bid params
+
+{: .table .table-bordered .table-striped }
+| Name  | Scope    | Description                     | Example  | Type      |
+|-------|----------|---------------------------------|----------|-----------|
+| `aid` | required | The source ID from Adtelligent. | `350975` | `integer` |
+
+
+
+### Test Parameters
+```
+    var adUnits = [
+
+      // Video instream adUnit
+      {
+        code: 'div-test-div',
+        sizes: [[640, 480]],
```

Comment:
this is a deprecated attribute. the recommendation is to put the player size in `mediaTypes.video.playerSize: [640, 480]`

Reply:
Confirmed!